addons/l10n_uy/models/account_move.py

```diff
@@ -0,0 +1,76 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    def _check_uruguayan_invoices(self):
+        """ Check that the UY invoices are vaid
+        * the partner of the invoice should have a valid identification type
+        * each invoice line should have one and only one vat tax
+        * export invoices should not have any vat tax.
+        """
+        uy_invs = self.filtered(lambda x: (x.company_id.country_id.code == 'UY' and x.l10n_latam_use_documents))
+        if not uy_invs:
+            return True
+        uy_invs.mapped('partner_id').check_vat()
+
+        uruguayan_vat_taxes = self.env['account.tax.group'].search([('l10n_uy_tax_category', '=', 'vat')])
+
+        # Check that we do not send any tax in export invoices
+        expo_cfes = uy_invs.filtered(
+            lambda x: int(x.l10n_latam_document_type_id.code) in [121, 122, 123])
+
+        # We check that there is one and only one vat tax per line
+        for line in (uy_invs - expo_cfes).mapped('invoice_line_ids').filtered(
+                lambda x: x.display_type not in ('line_section', 'line_note')):
+            vat_taxes = line.tax_ids.filtered(lambda x: x.tax_group_id in uruguayan_vat_taxes)
+            if len(vat_taxes) != 1:
+                raise ValidationError(_('Should be one and only one VAT tax per line. Verify lines with product "%s" (Id Invoice: %s)',
+                                      line.product_id.name, line.move_id.id))
+
+    def _get_l10n_latam_documents_domain(self):
+        """ We overwrite original domain to only show available active uruguayan document types
+        """
+        self.ensure_one()
+        domain = super()._get_l10n_latam_documents_domain()
+        if self.journal_id.company_id.account_fiscal_country_id.code == "UY":
+            codes = self.journal_id._l10n_uy_get_journal_codes()
+            if codes:
+                domain.extend([('code', 'in', codes), ('active', '=', True)])
```

Comment:
```suggestion
                domain.extend([('code', 'in', codes)])
```
That should not be necessary